.github/workflows/ROMROM-VERSION-BUMP.yaml

```
@@ -0,0 +1,69 @@
+        id: version
+        run: |
+          VERSION_LINE="${{ steps.current_version.outputs.current_version }}"
+          VERSION=$(echo $VERSION_LINE | cut -d'+' -f1)
+          BUILD_NUMBER=$(echo $VERSION_LINE | cut -d'+' -f2)
+
+          # patch 버전 증가 (1.0.0 → 1.0.1)
+          IFS='.' read -ra VERSION_PARTS <<< "$VERSION"
+          MAJOR=${VERSION_PARTS[0]}
+          MINOR=${VERSION_PARTS[1]}
+          PATCH=${VERSION_PARTS[2]}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)